typescript/solver/config.ts

```diff
@@ -0,0 +1,15 @@
+import dotenvFlow from "dotenv-flow";
+
+dotenvFlow.config();
+
+const MNEMONIC = process.env.MNEMONIC;
+const ORIGIN_SETTLER_ADDRESS = process.env.ORIGIN_SETTLER_ADDRESS;
+const ORIGIN_SETTLER_CHAIN_ID = process.env.ORIGIN_SETTLER_CHAIN_ID;
```

Comment:
I think it might be nice to turn this into configuration that is more similar to Hyperlane's relayer, i.e. have a dictionary of `chainName -> value` where value can be `contractAddresses`. And then there is a separate value for which chains or the topology similar to https://docs.hyperlane.xyz/docs/operate/relayer/message-filtering

Reply:
Something like https://github.com/hyperlane-xyz/hyperlane-registry/blob/main/deployments/warp_routes/AMPHRETH/arbitrum-ethereum-zircuit-addresses.yaml

Reply:
issue: https://github.com/BootNodeDev/intents-framework/issues/19